bios/lmrobin-bio.md

```diff
@@ -0,0 +1,4 @@
+## Lisa Robinson ...
+- from Connecticut shoreline
+- vacationed on Cape Cod last week
+- sandbar was awesome
```

Comment:
My daughter likes to collect rocks.....  luckily she seems to like small ones.....